cmd/helpers.go

```diff
@@ -223,7 +223,7 @@ func BuildNative(binaryName string, forceRebuild bool, buildMode string, project
 
 	if binaryName != "" {
 		// Alter binary name based on OS
-		switch runtime.GOOS {
+		switch projectOptions.Platform {
```

Comment:
This doesn't fully solve the issue. When creating a project on Windows, the `binaryname` option contains the `.exe`. We normally strip this out from non-windows platforms, as per the lines 231-235 below. As the docker binaries appear to have `-os-platform` as their suffix, I'm wondering if this check is now failing? Tried to print out the values but get nothing when the code path goes through `BuildDocker`

Reply:
I actually removed the word `.exe` from the code entirely and it STILL appears!  
I think the docker container might somehow be injecting it.   Unfortunately Windows is SO slow for me I might have to reboot into Windows itself on my laptop and see if I can get to the bottom of it.

Reply:
I was looking at the `build.sh` script and it only adds `extensions` for each platform (and `.exe` is definitely an extension).  

It's possible I wasn't running with the code I updated ?  I ran the `scripts/build.sh` to make sure it was updated but I don't believe it was working.    

When cross-compiling with Docker it should already be adding the appropriate extension so we shouldn't have to worry about it.

Reply:
Yes I know it does it for all platforms when executed on Windows!  I duplicated the problem on my machine (very slow machine).

Reply:
Yes I noticed it won't print the Docker commandline in verbose mode on Windows as well.  I added an additional `fmt.Printf` in front and that shows up but I didn't see any reason for the funny name since it wasn't passing that in to the docker run command at all.

I pushed commits so I could fetch them on my windows machine.  I'll let you know when i think its fixed.

Reply:
Is there anything I can do to help with this? 😃

Reply:
Dang it was a nightmare tracking down where the `xcc.exe` prefix was coming from!  It turns out projects created on Windows have that as their module name inside the `go.mod` file!

Reply:
Nice catch! Will review this today